composer.json

```diff
@@ -16,7 +16,7 @@
     ],
     "require": {
         "php": "^7.4|^8.0",
-        "tipoff/support": "^1.2.0"
+        "tipoff/support": "^1.2.6"
```

Comment:
These PR's removing the guarded property won't work unless we bump the dependency for tipoff/support